qa/qa-captain/bashrc

```diff
@@ -0,0 +1,1116 @@
+#! /bin/bash
```

Comment:
Are you using Bash specific features in this? If not, change this to `/bin/sh` as not all systems have bash and not all systems have bash at that location.

Reply:
I added this line to appease shellcheck really. So far none of the operating systems under test do not support Bash.

Reply:
The repo stanard is to use `/bin/sh`.

Reply:
This file is now inline with /bin/sh and shellcheck.